src/app/components/ATIAnalytics/types.ts

```diff
@@ -60,6 +63,32 @@ export interface PageData {
   title?: string;
 }
 
+export interface HomePageData {
+  pageType: string;
+  id?: string;
+  title: string;
+  curations: CurationData[];
+  description: string;
+  metadata: Pick<ATIData, 'analytics'>;
+}
```

Comment:
May be worth moving this out of here as its not too coupled to this component, but can be a follow up fix.